src/topp/InternalCalibration.cpp

```diff
@@ -169,6 +169,7 @@ class TOPPInternalCalibration :
     setValidFormats_("in", ListUtils::create<String>("mzML"));
     registerOutputFile_("out", "<file>", "", "Output file ");
     setValidFormats_("out", ListUtils::create<String>("mzML"));
+    registerInputFile_("executable", "<file>", "Rscript", "Path to the Rscript executable (default: 'Rscript').", false);
```

Comment:
Since the default is already in the default value (duh), I would remove the "(default: 'Rscript')" from the description.